.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+# Root
+* @smartcontractkit/foundations @smartcontractkit/core
+
+/evm @smartcontractkit/bix-framework @smartcontractkit/core
```

Let's add foundations here for now:
```suggestion
/evm @smartcontractkit/bix-framework @smartcontractkit/core @smartcontractkit/foundations
```

---

Done.